.github/workflows/test-command.yml

```diff
@@ -1,4 +1,4 @@
-name: integration-test
+name: Run Integration Test
```

Comment:
this one seems like kind of a monster.  I wonder if we can split these up more in the future.